config/locales/en.yml

```
@@ -615,7 +615,7 @@ en:
           support_desc_html: "The WCA website supports %{two_factor_link} using a one-time password:"
           #context: describes the 2FA options
           options:
-            app: "generated by a dedicated 2FA application on your smartphone (such as Authy or Google Authenticator for instance)."
+            app: "generated by a dedicated 2FA application on your smartphone (such as Authy, Google Authenticator or iOS Passwords)."
```

```suggestion
            app: "generated by a dedicated 2FA application on your smartphone (such as Authy, Google Authenticator, or iOS Passwords)."
```
Not sure if this is consistent with other WST-related things, but added an Oxford comma: